templates/classpath-jars-pv.yaml

```diff
@@ -48,3 +49,4 @@ spec:
     volumeAttributes:
       path: {{ include "palisade.deployment.path" . }}
   {{- end }}
+  {{- end }}
```

Comment:
Can the indentation be sorted so that the `{{- end }}` is inline with the `{{- if }}`?

Reply:
Thanks, got it,